### ISSUE TITLE
Move Solr files to Broadsea-Solr

### ISSUE THREAD

**Issue by alondhe** [OHDSI/WebAPI#2165]
Currently, we have the Solr files maintained in this repo, as a reference for users to be able to stand up Solr search for their WebAPI instance. 

As there are quite a bit of configuration changes to make, along with just the Solr install itself, I am thinking it might be best to shift these files to a separate repo for more direct maintenance and easier usage. I've made a PR for the Broadsea-Solr repo (https://github.com/OHDSI/Broadsea-Solr/pull/1) to address this. 

Now, I realize that not everyone can use Docker, but the solr/ folder within this proposed PR would be something even non-Docker users could grab and run with. If we agree on this idea, I can shift the parameter tokenization steps from being hardcoded in the Dockerfile to a separate shell script that Docker and non-Docker users can leverage.

So TLDR: remove Solr files from WebAPI and direct users (assuming we merge my PR) to the Broadsea-Solr repo instead.

**Comment by alondhe** [comment]
Upon discussion -- Solr in WebAPI now means OMOP Vocab *and*, in 2.14, advanced concept set searching. So Broadsea-Solr will need to support multiple cores for different purposes and different lifecycles. I think for now, changing the config files for the omop vocabulary to be the parameterized ones I established will be a big benefit.
